subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -492,11 +494,17 @@ static int wifi_start_roaming(uint32_t mgmt_request, struct net_if *iface,
 		}
```

Comment:
Please remove `wifi_mgmt_api == NULL` for 11k case

Reply:
done